build.gradle

```diff
@@ -119,8 +119,8 @@ tasks.named('sonarqube').configure {
 }
 
 dockerCompose {
-    useComposeFiles = ["docker/docker-compose.yaml"]
-    startedServices = ['sge-container']
+    useComposeFiles = ["docker/slurm/docker-compose.yml"]
```

Comment:
I believe, that we shouldn't just replace the target files with Slurm-related ones, but implement some branching. 
Let's say we want to rely on some `GRID_ENGINE_API_TARGET_ENGINE` env var and:
- set up SGE environment in case of `SGE` value
- set up Slurm environment for Slurm
- throw some exception in case of unknown value